README.md

```diff
@@ -62,70 +84,122 @@ highlighted](https://i.imgur.com/XN7xdKF.png)
+distance fraction.
 
 The user agent may trade off precision for efficiency in the computation of
 LS scores.  It is intended that the LS score have a correspondence to the
 perceptual severity of the instability, but not that all user agents produce
 exactly the same LS scores for a given page.
 
+### Performance API
+
+Animation frames with non-zero LS scores will notify a registered
+[PerformanceObserver](https://w3c.github.io/performance-timeline/#the-performanceobserver-interface).
+The observer's callback receives one or more `LayoutShift` entries:
+
+```idl
+interface LayoutShift : PerformanceEntry {
+    readonly attribute double value;
+    readonly attribute boolean hadRecentInput;
+    readonly attribute DOMHighResTimeStamp lastInputTime;
+};
+```
+
+The entry's `value` attribute is the LS score.  Its
+[entryType](https://w3c.github.io/performance-timeline/#dom-performanceentry-entrytype)
+attribute is `"layout-shift"`.
+
+The `hadRecentInput` and `lastInputTime` attributes are described in
+[Recent Input Exclusion](#Recent-Input-Exclusion).
+
 ### Cumulative Scores
 
-The user agent can compute a *document cumulative layout shift* (DCLS) score as
-the sum of the document's LS scores for each animation frame that has occurred
+The user agent can compute a **document cumulative layout shift** (DCLS) score
+as the sum of the document's LS scores for each animation frame that has occurred
 during the browsing session.  The DCLS score is 0 when the document begins
 loading, and grows whenever layout instability occurs.  The DCLS score does not
 account for layout instability inside descendant browsing contexts, such as
 those created by `<iframe>` elements.
 
-The user agent can compute a *cumulative layout shift* (CLS) score for a
+The user agent can compute a **cumulative layout shift** (CLS) score for a
 [top-level browsing context](https://html.spec.whatwg.org/multipage/browsers.html#top-level-browsing-context)
 by summing the LS scores of the top-level browsing context to the weighted LS
 scores of its descendant browsing contexts.  In performing this aggregation,
 the LS score of a layout shift in an `<iframe>` should be weighted by the
 fraction of the top-level viewport the `<iframe>` occupies at the time the
 layout shift occurs.
 
-### Performance API
+The DCLS and CLS scores are not directly exposed by the [Performance API](#Performance-API),
+but we hope to make it easy for developers to construct these from the LS scores.
 
-Animation frames with non-zero layout shift (LS) scores will notify a registered
-[PerformanceObserver](https://w3c.github.io/performance-timeline/#the-performanceobserver-interface).
-The observer's callback receives one or more `LayoutShift` entries:
+### Recent Input Exclusion
 
-```idl
-interface LayoutShift : PerformanceEntry {
-    readonly attribute double value;
-};
-```
+In calculating DCLS and CLS scores, developers and user agents may wish to
```

Comment:
This is geared towards developers... maybe just developers here instead of developers and user agents?

Reply:
I think the fact that the user agent may be computing these scores independently is important to highlight as providing hints like hadRecentInput is an important part of allowing the developer to match the user agent's algorithm